### PR TITLE
{math}[foss/2020a] Transformers v4.2.1 w/ Python 3.8.2

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node309.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/e6a1aed11772800d528b54efef1fecf9 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/32d02118887d984c5b440ead08b0894b for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/t/Transformers/Transformers-4.2.1-foss-2020a-Python-3.8.2.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'Transformers'
+version = '4.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/facebookresearch/detectron://github.com/huggingface/transformers'
```
Comment (verdurin) [inline easybuild/easyconfigs/t/Transformers/Transformers-4.2.1-foss-2020a-Python-3.8.2.eb:7]: @smoors the `homepage` URL looks wrong - pasting error?

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/Transformers/Transformers-4.2.1-foss-2020a-Python-3.8.2.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/t/Transformers/Transformers-4.2.1-foss-2020a-Python-3.8.2.eb]: indeed, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12032/commits/3aa1fd85d53d997540751e1e07b500849cb001f0

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @smoors!

**Pull request merged**